src/main/java/in/org/projecteka/hiu/consent/model/Consent.java

```diff
@@ -15,14 +15,15 @@
 import java.util.List;
 import java.util.TimeZone;
 
-import static in.org.projecteka.hiu.consent.model.consentmanager.Frequency.ZERO_HOUR;
+import static in.org.projecteka.hiu.consent.model.consentmanager.Frequency.ONE_HOUR;
 
 @JsonIgnoreProperties(ignoreUnknown = true)
 @AllArgsConstructor
 @Builder
 @NoArgsConstructor
 @Data
 public class Consent {
+    private static final String CALL_BACK_URL = "http://hiu-dev.projecteka.in/";
```

Comment:
This should not be hardcoded. at the least, it should be an application property. I could possibly think of the callback URL to be dynamic, but probably later